projects/adxrs290-pmdz/src/examples/examples_src.mk

```diff
@@ -8,6 +8,7 @@ endif
 ifeq (y,$(strip $(IIO_TRIGGER_EXAMPLE)))
 TINYIIOD=y
 CFLAGS += -DIIO_TRIGGER_EXAMPLE=1
+CFLAGS +=-DIIO_IGNORE_BUFF_OVERRUN_ERR
```

Comment:
Should this be the default behavior?

Reply:
for adxrs yes, because the sampling period cannot be changed and will cause an error overrun error if the flag is not set.